department-of-veterans-affairs/abd-vro
# Update Lookup Table Versions

## What was the problem?
This PR updates the lookup table versions for the condition dropdown and diagnostic code tables.  Additionally, it updates the autosuggestion list for use in logging metrics.

Associated tickets or Slack threads:
- [#509 - Make updates to autosuggestions list and lookup tables](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/509)
- [Slack Thread](https://dsva.slack.com/archives/C04QKNK4J0H/p1724958840582939)

## How does this fix it?[^1]
This adds the correct version of csv and updates the lookup table. 

## How to test this PR
- from `cc-app` run `pytest`

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
